### DISCUSSION TITLE
Practica Abogabot - Terminado

### DISCUSSION THREAD

**Discussion by Will0220** [Launch-X-Latam/MisionFrontEnd#248] [General]
Hola chicos, paso por aqui para dejarles mi propuesta de la práctica de ABOGABOT, quisiera que pasarán a darle una revisada y opinen que les pareció. 
Agradecería mucho que me dieran feedback para mejorar!  Gracias chicos!

https://github.com/Will0220/Abogabot

**Comment by alatorre-sebastian** [comment]
Te quedo genial, en que hiciste el UI/UX ?

**Reply by Will0220** [reply]
Hola!!, muchas gracias. El UX lo hice en Balsamiq: https://balsamiq.cloud/ y el UI lo hice en Figma: figma.com

**Comment by CarlosBinary** [comment]
¡Sorprendente!
